src/main/java/org/isf/menu/service/UserIoOperationRepository.java

```diff
@@ -62,10 +71,16 @@ public interface UserIoOperationRepository extends JpaRepository<User, String> {
 	@Query(value = "update User set lastLogin=:lastLoggedIn where userName=:id")
 	void setLastLogin(@Param("lastLoggedIn") LocalDateTime lockTime, @Param("id") String id);
 
-	@Query("select count(u) from User u where active=1")
+	@Query("select count(u) from User u where active=1 and deleted!=true")
 	long countAllActiveUsers();
 
-	@Query("select count(g) from UserGroup g where active=1")
+	@Query("select count(u) from User u where active=1 and deleted=:deleted")
+	long countAllActiveUsersByDeleted(@Param("deleted") boolean deleted);
+
+	@Query("select count(g) from UserGroup g where active=1 and deleted!=true")
```

Comment:
```suggestion
	@Query("select count(g) from UserGroup g where active=1 and deleted==false")
```
Why use `!=true`?   What am I missing?   Especially since if `false` is passed to `countAllActiveGroupsByDeleted()` (see line 83) the condition is `and deleted=false`